test/index.spec.js

```diff
@@ -76,3 +76,29 @@ describe('multihashing', () => {
     })
   })
 })
+
+describe('validate', () => {
+  it('true on pass', done => {
+    multihashing(Buffer.from('test'), 'sha2-256', (err, hash) => {
+      if (err) throw err
```

Comment:
I don't know if that matters, but other test cases return `return done(err)` in case of an error instead of throwing.

Besides that a common pattern in other test suites is to use `expect(err).to.not.exist()` instead.